src/components/fields/MultiSchemaField.js

```diff
@@ -81,6 +81,10 @@ class AnyOfField extends Component {
       }
     }
 
+    if(this.state) {
+      return this.state.selectedOption || 0
```

Comment:
Is it possible that selectedOption can be `false` or `null`?

Reply:
nope